gems/pending/appliance_console.rb

```diff
@@ -38,7 +38,7 @@
 
 [:INT, :TERM, :ABRT, :TSTP].each { |s| trap(s) { raise MiqSignalError } }
 
-RAILS_ROOT    = Pathname.new("#{ROOT}/vmdb")
+RAILS_ROOT    = Pathname.new("#{ROOT}")
```

Comment:
Not that I've heard ...
I'm judging by https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/etc/default/evm_bundler#L5 and https://github.com/ManageIQ/manageiq-appliance-build/blob/master/kickstarts/partials/post/source_setup.ks.erb#L13
So we are still setting `BUNDLE_GEMFILE` to `/var/www/miq/vmdb/Gemfile` and checking the manageiq repo out to `/var/www/miq/vmdb` and `Gemfile` is still in the root of the manageiq repo.  Not sure if there are plans to change that, but nothing has happened yet.

I think https://github.com/ManageIQ/manageiq/pull/6208/files#diff-16bd2bb3c0abfe4c0381896614e7b987R14 is wrong, but isn't getting set because `ENV['BUNDLE_GEMFILE']` is set by `/etc/default/evm_bundler`